Themes update docs (#883)
telerik/blazor-docs: _contentTemplates/common/general-info.md

#cdn
You can reference the built-in Telerik assets from a cloud CDN instead of a local resource on your server.

> Telerik Blazor CDN distributes the most commonly used swatches of each base theme. Here you can find a list of their CDN URLs - [Telerik Blazor CDN]({%slug swatch-distribution%}#telerik-blazor-cdn).

Reviewer: Don't we provide ALL swatches for each base theme?

```suggestion
The Telerik Blazor CDN distributes the most common swatches of each base theme. Check out the [Telerik Blazor CDN URLs]({%slug swatch-distribution%}#telerik-blazor-cdn).
```
